content/applications/finance/accounting/reporting/overview/tax_carryover.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Tax carryover
+=============
+
+When performing tax reports, the :guilabel:`tax carryover` feature allows carrying amounts from one
+period to the other without creating new entries.
+
+It has been created to respond to legal requirements in some locations where amounts must be
+transferred from period to period (for example, because the total of the line is negative).
```

Comment:
It has been created to meet the legal requirements of specific locations, where amounts must be transferred from period to period (for instance, when the total of a line is negative).